packages/core/src/client.test.ts

```diff
@@ -1401,7 +1402,7 @@ describe('Client', () => {
 
     const fetch = mockFetch(200, {});
     const client = new MedplumClient({ fetch });
-    const promise = client.createBinary('Hello world', undefined, ContentType.TEXT, onProgress);
+    const promise = client.createBinary(Readable.from('Hello World'), undefined, ContentType.TEXT, onProgress);
```

Comment:
Could move this to a separate test, if deemed necessary.
Could also do a more fine-grained test of progress tracking now that we can directly mock readable.

Reply:
I think having both tests is good, just so we're hitting both paths through the codebase 👍